ReactWindows/ReactNative/Chakra/Executor/NativeJavaScriptExecutor.cs

```diff
@@ -104,6 +115,24 @@ public void RunScript(string script, string sourceUrl)
             }
         }
 
+        private async Task<bool> EnsureSerializedScriptAsync(string scriptFile, string binPath)
+        {
+            var localFolder = ApplicationData.Current.LocalFolder;
+
+            try
+            {
+                var scriptItem = await localFolder.GetItemAsync(Path.GetFileName(scriptFile));
```

Comment:
Also, there is definitely no guarantee that the scriptFile is in the localFolder.